google-cloud-datastore/src/main/java/com/google/cloud/datastore/testing/LocalDatastoreHelper.java

```diff
@@ -151,21 +157,21 @@ public LocalDatastoreHelper build() {
     }
   }
 
-  LocalDatastoreHelper(Builder builder) {
+  private LocalDatastoreHelper(Builder builder) {
```

Comment:
you have 2 ctors doing the same things:
- private LocalDatastoreHelper(Builder builder) {
- private LocalDatastoreHelper(double consistency, int port) {

please remove the second one and update code to use the builder approach

Reply:
i think we need to confirm with Benwhite before removing

Reply:
These ctors are private, so they can be removed without breaking API.
If you want to keep them both, please call from another to avoid code duplication.

Reply:
done